server/src/projects/projects.controller.ts

```diff
@@ -62,6 +62,8 @@ import { INVOICE_ATTRS } from '../invoices/invoices.attrs';
       ref: 'contactid',
       attributes: [
         ...CONTACT_ATTRS,
+
+        'is_nycid_email_registered',
```

Comment:
Because it's not a real CRM attr

Reply:
@godfreyyeung should never never never have "virtual" properties or things like relationships in the attrs because those are used for CRM API queries.